tests/e2e/Cargo.toml

```diff
@@ -16,3 +16,4 @@ env_logger = "0.9.0"
 log = "0.4.17"
 serde_json = "1.0.81"
 tokio = { version = "1.18.1", features = [] }
+avalanche-proto = { version = "0.15.1" }
```

Comment:
nit: can we keep imports alphabetical ordered and only add the feature we use for the import `feature = ["rpcpb"]`